test/drenv/__main__.py

```
@@ -37,6 +38,14 @@
 def main():
     args = parse_args()
     configure_logging(args)
+    git_info = git.info()
```

Ruff (0.16.1)

[error] Undefined name `git`

(F821)